### PR TITLE
solving issue 2544... changing team leadership

### LINKED ISSUE #2544: Update the Current Project Team Section for HfLA Website Project
### Overview
We need to update the project team section of the HFLA Website project page to properly reflect the current members of the leadership team.

### Details
We need to remove Mathew Cacho, Sihem Gourou, Kristine Eudey and add Tomas Gonzalez, Saumil Dhankar, Isaac Cruz, and Simone Campbell on the Website team project page leadership section.

### Action Items
- [x] Remove Mathew Cacho and add Tomas Gonzalez, Saumil Dhankar as Technical Leads on the HfLA Team. Change the links accordingly as well.
    - **Edit Technical Lead section** under `_projects/website.md`

Changes required in the following section:
```
  - name: Matthew Cacho
    role: Technical Lead
    links:
      slack: 'https://hackforla.slack.com/team/U0262GA4DD2'
      github: 'https://github.com/macho-catt'
    picture: https://avatars.githubusercontent.com/macho-catt
```

- [x] Remove Sihem Gourou as Product Manager on the HfLA Team
    - **Remove Product Manager section** under `_projects/website.md`

Remove the following section:
```
  - name: Sihem Gourou
    role: Product Manager
    links:
      slack: 'https://hackforla.slack.com/team/U021GM545UG'
      github: 'https://github.com/Sihemgourou'
    picture: https://avatars.githubusercontent.com/Sihemgourou
```

- [x] Remove Kristine Eudey and add Isaac Cruz as UX Design Lead on the HfLA Team. Change the links accordingly as well.
    - **Edit UX Design Lead section** under `_projects/website.md`

Changes required in the following section:
```
  - name: Kristine Eudey
    role: UX Design Lead
    links:
      slack: 'https://hackforla.slack.com/team/U0252CHL678'
      github: 'https://github.com/kristine-eudey'
    picture: https://avatars.githubusercontent.com/kristine-eudey
```

- [x] Add Simone Campbell as UX Research Lead on the HfLA Team
    - **Add UX Research Lead section** under `_projects/website.md`

Add the following section:
```
  - name: Simone Campbell
    role: UX Research Lead
    links:
      slack: 'https://hackforla.slack.com/team/U0260MKR5EJ'
      github: 'https://github.com/sacamp'
    picture: https://avatars.githubusercontent.com/sacamp
```

### Resources/Instructions
- Directory to find the page in once you have it in your IDE: `_projects/website.md`
- [Projects page](https://www.hackforla.org/projects/)
- [Website page](https://www.hackforla.org/projects/website)
- Click on the filename below to see the source file code:
https://github.com/hackforla/website/blob/gh-pages/_projects/website.md?plain=1

### PR DESCRIPTION
**Pull request by Georgema20**
Fixes #2544 

### What changes did you make and why did you make them ?

I removed Mathew Cacho, Sihem Gourou, Kristine Eudey and add Tomas Gonzalez, Saumil Dhankar, Isaac Cruz, and Simone Campbell on the Website team project page leadership section.

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

Screen Shot 2021-12-22 at 3 27 10 PM

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
Screen Shot 2021-12-24 at 9 20 22 PM

</details>

### CODE REVIEW

File: _projects/website.md [hunk 1]
```diff
@@ -1,51 +1,57 @@
...
+    picture: https://avatars.githubusercontent.com/mmaecruz
+  - name: Simone Campbell
+    role: UX Research Lead
     links:
-      slack: 'https://hackforla.slack.com/team/U021GM545UG'
-      github: 'https://github.com/Sihemgourou'
-    picture: https://avatars.githubusercontent.com/Sihemgourou
+      slack: "https://hackforla.slack.com/team/U0260MKR5EJ"
+      github: "https://github.com/sacamp"
+    picture: https://avatars.githubusercontent.com/sacamp
```
Comment (JessicaLucindaCheng) [inline _projects/website.md:45]: Needs single quotes around https://avatars.githubusercontent.com/sacamp

File: _projects/website.md [hunk 2]
```diff
@@ -1,51 +1,57 @@
...
     role: UX Design Lead
     links:
-      slack: 'https://hackforla.slack.com/team/U0252CHL678'
-      github: 'https://github.com/kristine-eudey'
-    picture: https://avatars.githubusercontent.com/kristine-eudey
-  - name: Sihem Gourou
-    role: Product Manager
+      slack: "https://hackforla.slack.com/team/U02HXLZ9JGN"
+      github: "https://github.com/mmaecruz"
+    picture: https://avatars.githubusercontent.com/mmaecruz
```
Comment (JessicaLucindaCheng) [inline _projects/website.md:39]: Needs single quotes around https://avatars.githubusercontent.com/mmaecruz

**Comment by Georgema20** [comment]
@JessicaLucindaCheng I do not see your comments anymore, but I saw your email. You said all the pictures need "" around them? True?

**Review by JessicaLucindaCheng — CHANGES_REQUESTED** [review]
- All double quotes need to be changed to single quotes.
- I think the contact information for Isaac Cruz is incorrect. Isaac's GitHub is https://github.com/IsaacDesigns and Slack id is U0255408P8Q.
- Also, please add your pull request to the project board. Thank you.

**Comment by JessicaLucindaCheng** [comment]
> @JessicaLucindaCheng I do not see your comments anymore, but I saw your email. You said all the pictures need "" around them? True?

Sorry, that was my mistake. The pictures don't need quotes. The pictures are fine as is. See my review above requesting changes.

**Comment by Georgema20** [comment]
@JessicaLucindaCheng  making all changes right now.

**Comment by Georgema20** [comment]
@JessicaLucindaCheng 

Do you mean every single double quote needs to be changed to single quote? Before in the original, there were double quotes in some things prior to me.

**Comment by JessicaLucindaCheng** [comment]
@Georgema20 I think your IDE is automatically changing the quotes to double quotes. If you look at the Files changed tab, you can see the single quotes on each line are changed to double quotes. So, please change all the double quotes back to single quotes.

**Comment by Georgema20** [comment]
@JessicaLucindaCheng  DOne! I realized that it was an extension I had on github that was doing that... my bad. Turned that off so it won't be a problem in the future.

**Review by JessicaLucindaCheng — CHANGES_REQUESTED** [review]
@Georgema20 Thanks for making the requested changes! The code looks good now.

Could you update the image for the "Visuals after changes are applied" in your initial post/comment? I see something different now.

**Comment by Georgema20** [comment]
@JessicaLucindaCheng Done!

**Review by JessicaLucindaCheng — APPROVED** [review]
@Georgema20 Looks good to me! Good job!

**Review by SAUMILDHANKAR — APPROVED** [review]
@Georgema20 Thanks for working on this issue. Changes look good in my environment.

**Pull request merged**